Pipfile

```diff
@@ -5,6 +5,7 @@ name = "pypi"
 
 [dev-packages]
 pytest = "*"
+mypy = "*"
```

Comment:
Nice! When I tried to run it locally I've got an error and I had to install the `typing_extensions` package. Maybe it would be good to add that as a dev dependency as well.

Reply:
Weird! Pip continues to be a mysterious black box. ;)